apps/condo/bin/delete-contacts-by-organization-id.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path')
+const { find } = require('@condo/keystone/schema')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { isEmpty } = require('lodash/isEmpty')
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+
+class DeleteContactsWithOrganizationId{
+    constructor (organizationIds) {
+        this.organizationIds = organizationIds
+        this.context = null
+    }
+
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+        this.context = await keystone.createContext({ skipAccessControl: true })
+    }
+
+    async findContacts () {
+        return await find('Contact', {
+            deletedAt: null,
+            organization: {
+                id_in: this.organizationIds,
+            },
+        })
+    }
+
+    async deleteContacts () {
+        const contacts = await this.findContacts()
+
+        console.info(`[INFO] Following contacts will be deleted: [${contacts.map(contact => `'${contact.id}'`).join(', ')}]`)
+
+        if (isEmpty(contacts)) return
+
+        for (const contact of contacts) {
+            await Contact.softDelete(this.context, contact.id, {
+                dv: 1,
+                sender: { dv: 1, fingerprint: 'deleteIncorrectContactsScript' },
```

Comment:
Why are these contacts incorrect? These contacts are regular, at least according to the find-conditions.
I'd like to prevent misunderstandings in the future :)
```suggestion
                sender: { dv: 1, fingerprint: 'deleteContactsByOrganizationsIdsScript' },
```